src/Symfony/Component/Serializer/Command/DebugCommand.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Command;
+
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Dumper;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Finder\Exception\DirectoryNotFoundException;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Serializer\Mapping\ClassMetadataInterface;
+use Symfony\Component\Serializer\Mapping\Factory\ClassMetadataFactoryInterface;
+
+/**
+ * A console command to debug Serializer information.
+ *
+ * @author Loïc Frémont <lc.fremont@gmail.com>
+ */
+#[AsCommand(name: 'debug:serializer', description: 'Display serialization information for classes')]
+class DebugCommand extends Command
+{
+    public function __construct(private readonly ClassMetadataFactoryInterface $serializer)
+    {
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->addArgument('class', InputArgument::REQUIRED, 'A fully qualified class name or a path')
+            ->addOption('show-all', null, InputOption::VALUE_NONE, 'Show all classes even if they have no serialization information')
+            ->setHelp(<<<'EOF'
+The <info>%command.name% 'App\Entity\Dummy'</info> command dumps the serializer groups for the dummy class.
+
+The <info>%command.name% src/</info> command dumps the serializer information for the `src` directory.
+EOF
+            )
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $class = $input->getArgument('class');
+
+        if (class_exists($class)) {
+            $this->dumpSerializerDataForClass($input, $output, $class);
+
+            return 0;
```

Comment:
I'm not sure why we don't use the constant everywhere.
```suggestion
            return Command::SUCCESS;
```